instrumentation/opentelemetry-instrumentation-starlette/src/opentelemetry/instrumentation/starlette/__init__.py

```diff
@@ -344,18 +359,20 @@ def _get_route_details(scope):
     return route
 
 
-def _get_default_span_details(scope):
-    """
-    Callback to retrieve span name and attributes from scope.
+def _get_default_span_details(
+    scope: dict[str, Any],
+) -> tuple[str, dict[str, Any]]:
```

Comment:
The `line-lenght` limit is too low for this project.

Reply:
Maybe we can discuss this project wide in the next Python SIG call?

Reply:
When's that?

Reply:
9AM PST every Thursday. See [calendar](https://calendar.google.com/calendar/u/0/embed?src=c_2bf73e3b6b530da4babd444e72b76a6ad893a5c3f43cf40467abc7a9a897f977@group.calendar.google.com) for details.